nixos/tests/lorri/default.nix

```diff
@@ -0,0 +1,26 @@
+import ../make-test-python.nix {
+  machine = { pkgs, ... }: {
+    imports = [ ../../modules/profiles/minimal.nix ];
+    environment.systemPackages = [ pkgs.lorri ];
+  };
+
+  testScript = ''
+    # Copy files over
+    machine.succeed(
+        "cp '${./fake-shell.nix}' shell.nix"
+    )
+    machine.succeed(
+        "cp '${./builder.sh}' builder.sh"
+    )
+
+    # Start the daemon and wait until it is ready
+    machine.execute("lorri daemon > lorri.stdout 2> lorri.stderr &")
```

Comment:
At least according to previous discussions, we don't really want to run lorri as the `root` user, do we?

I'd suggest to import `./common/user-account.nix`, and run it as alice or bob.

We also don't test the socket activation at all. I propose starting and forking the daemon manually should be done for `alice`, and `bob` should rely on systemd socket activation. WDYT?

Reply:
My thinking:
```
no test < current test < what you're proposing
```
Unfortunately I can't get your proposal to work! So I'm inclined to merge now with the test as-is, and improve on the test in a follow-up.

Reply:
Captured this task here: https://github.com/target/lorri/issues/218